### PR TITLE
GH-531 Add info about /jira webhook

### PR DESCRIPTION
**Pull request by GrigalashviliT**
#### Summary
 jira plugin config UI changed and `/jira webhook` added there
jira webhook URL added in outputs of `/jira install` and `/jira uninstall`

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-plugin-jira/issues/531

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -88,6 +88,6 @@
         "default": false
       }
     ],
-    "footer": "Use this webhook URL format to [configure the Jira integration.](https://github.com/mattermost/mattermost-plugin-jira/blob/master/readme.md)  `https://SITEURL/plugins/jira/api/v2/webhook?secret=WEBHOOKSECRET`"
+    "footer": "Use `/jira webhook` to see fully expanded URL to [configure the Jira integration.](https://github.com/mattermost/mattermost-plugin-jira/blob/master/readme.md)"
```
Comment (Francois-D) [inline plugin.json:91]: 2/5  - I recommend updating this section to inform the user to run the /jira webhook command inside of a channel.  The example URL is great to show the webhook URL and the importance of  the Webhook-secret.   

https://mattermost.gitbook.io/plugin-jira/setup/configuration#step-4-required-setup-webhooks-from-jira

Perhaps even a ticket should be created if a secret is not generated the webhook displays an error?

**Comment by jfrerich** [comment]
@aaronrothschild, ping for review 😃

**Comment by DHaussermann** [comment]
I did review these change also. No issues found. Thanks @GrigalashviliT! This makes for a huge visibility improvement.

File: plugin.json (hunk 1 above)
Comment (jaydeland) [reply plugin.json]: @Francois-D Thanks for the effort on this and https://github.com/mattermost/mattermost-plugin-jira/pull/525

**Comment by hanzei** [comment]
@levb @jfrerich Are these changes read to be merged?

**Pull request merged**